HVZ/HVZ/api/views.py

```diff
@@ -52,6 +59,9 @@ def form_valid(self, form):
             body = form.cleaned_data['body']
             recipient_title = form.cleaned_data['recipient']
 
```

Comment:
The commented sender "mod@claremonthvz.org" will be used for the actual mods of HvZ